### PR TITLE
Adding basic unit tests

### PR DESCRIPTION
**Pull request by leeyun-amzn**
Signed-off-by: Annie Lee <leeyun@amazon.com>

### Description
- Adding a few basic unit tests.
-  Update jest config file to exclude some files. This makes the coverage report more accurate when running `yarn run jest: test --coverage` locally.
- Simplify the code in `VisualGraph.js` to make code more readable.
- Set jest to run under UTC timezone in`package.json` file.
 
### Issues Resolved
Part of #129
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by AWSHurneyt — APPROVED** [review]

File: public/pages/CreateMonitor/components/VisualGraph/__snapshots__/VisualGraph.test.js.snap [hunk 1]
```diff
@@ -1,11 +1,5 @@
...
-  style="display:flex;align-items:center;justify-content:center;height:450px"
->
-  <div>
-    There is no data for the current selections.
-  </div>
-</div>
-`;
+exports[`VisualGraph renders 1`] = `null`;
+
+exports[`VisualGraph renders with bucket level monitor 1`] = `null`;
```
Comment (qreshi) [inline public/pages/CreateMonitor/components/VisualGraph/__snapshots__/VisualGraph.test.js.snap:5]: Are these snapshots supposed to be null?

File: public/pages/Dashboard/containers/Dashboard.test.js [hunk 2]
```diff
@@ -104,4 +193,26 @@ describe('Dashboard', () => {
     expect(wrapper.instance().state.alerts[0].alert_history).toStrictEqual([]);
     expect(wrapper.instance().state.alerts[0].error_message).toBe('');
   });
+
+  test('able to select single alert in flyout', () => {
```
Comment (qreshi) [inline public/pages/Dashboard/containers/Dashboard.test.js:197]: Looks like this test is just testing the render situation for a flyout when `perAlertView` is not set.

Comment (leeyun-amzn) [reply public/pages/Dashboard/containers/Dashboard.test.js]: Yes, I found out this is incomplete and will need some time to figure out the right way to perform the user events. Skipping this test with added TODO for now.

File: public/pages/CreateMonitor/components/VisualGraph/__snapshots__/VisualGraph.test.js.snap (hunk 1 above)
Comment (leeyun-amzn) [reply public/pages/CreateMonitor/components/VisualGraph/__snapshots__/VisualGraph.test.js.snap]: No, I found out that there were additional data needed in order to let the snapshot show up. They should be good now.

**Review by qreshi — APPROVED** [review]

**Review by AWSHurneyt — APPROVED** [review]

**Pull request merged**